moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_frame.cpp

```diff
@@ -51,8 +51,6 @@
 
 #include "ui_motion_planning_rviz_plugin_frame.h"
 
-#include <tf2_ros/buffer.h>
```

Comment:
This header is likely implicitly included, but now that we're explicitly declaring buffers in this file, I think it makes sense to keep the header.